Source/WebCore/PlatformHaiku.cmake

```diff
@@ -232,6 +237,7 @@ if (ENABLE_VIDEO)
     )
 endif ()
 
+# TODO: WTF_USE_3D_GRAPHICS seems to have disappeared
```

Comment:
Yes, USE_3D_GRAPHICS is always enabled now so this can be removed or always enabled (whichever lets you compile things).

Reply:
I see this is removed in the next commit, so that's already done :)

Reply:
Already done :)

Reply:
Ah, just saw your comment above. GH didn't show it earlier